### PR TITLE
feat: GEOT-814: add girls faces

### PR DESCRIPTION
**Pull request by ra8ga**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "dependencies": {
     "@datorama/akita": "^5.3.0",
+    "@types/react-avatar-editor": "^10.3.6",
```
Comment (ra8ga) [inline package.json:8]: bacause of 

> @gtms/app-andrew
$ tsc --noEmit -p ./tsconfig.json
../ui/ImageEditor/index.tsx:2:31 - error TS7016: Could not find a declaration file for module 'react-avatar-editor'. '/Users/rafal/Desktop/gtms-frontend/node_modules/react-avatar-editor/dist/index.js' implicitly has an 'any' type.
  Try `npm install @types/react-avatar-editor` if it exists or add a new declaration (.d.ts) file containing `declare module 'react-avatar-editor';`

2 import ReactAvatarEditor from 'react-avatar-editor'

File: packages/ui/ImageEditor/index.tsx [hunk 2]
```diff
@@ -169,17 +169,19 @@ export const ImageEditor: FC<{
                     return
                   }
 
-                  editorRef.current.getImageScaledToCanvas().toBlob((blob) => {
-                    if (!blob) {
-                      return
-                    }
+                  editorRef.current
+                    .getImageScaledToCanvas()
+                    .toBlob((blob: Blob | null) => {
```
Comment (ra8ga) [inline packages/ui/ImageEditor/index.tsx:174]: is this typing ok? blob: Blob | null

bacause:

../ui/ImageEditor/index.tsx:172:70 - error TS7006: Parameter 'blob' implicitly has an 'any' type.

172                   editorRef.current.getImageScaledToCanvas().toBlob((blob) => {

Comment (mariusz-kabala) [reply packages/ui/ImageEditor/index.tsx:174]: po co ci null tam?

Comment (ra8ga) [reply packages/ui/ImageEditor/index.tsx]: > @gtms/app-andrew
$ tsc --noEmit -p ./tsconfig.json
../ui/ImageEditor/index.tsx(174,29): error TS2345: Argument of type '(blob: Blob) => void' is not assignable to parameter of type 'BlobCallback'.
  Types of parameters 'blob' and 'blob' are incompatible.
    Type 'Blob | null' is not assignable to type 'Blob'.
      Type 'null' is not assignable to type 'Blob'.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed.
Exit code: 1
Command: /usr/local/bin/node
Arguments: /opt/yarn-v1.22.0/lib/cli.js run check:ts
Directory: /var/lib/jenkins/workspace/(GTMS Frontend) Run checks on Pull Request/packages/app-andrew
Output:

info Visit https://yarnpkg.com/en/docs/cli/workspaces for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
[Pipeline] }
[Pipeline] // script
Post stage
[Pipeline] script
[Pipeline] {
[Pipeline] sh
Warning: A secret was passed to "sh" using Groovy String interpolation, which is insecure.
		 Affected argument(s) used the following variable(s): [GITHUB_API_KEY]
		 See https://jenkins.io/redirect/groovy-string-interpolation for details.
+ curl -s -X POST -d 

screencapture-jenkins-kabala-tech-job-GTMS-Frontend-Run-checks-on-Pull-Request-2203-console-2021-09-06-17_33_55

**Pull request closed without merging**